lib/tasks/example_data.rake

```diff
@@ -206,5 +192,10 @@ namespace :example_data do
         end
       end
     end
+
+    # We don't want all trainees having the same update date on the index page
+    Audited.audit_class.all.each do |audit|
```

Comment:
Should we make the Audit match the creation/update date on the trainee? I suppose the question there really is should we randomise the dates on the trainee?

Reply:
I've updated the code so that the trainee timestamps match the audit log.